docs/content/mobile/android/local-deploy.md

```diff
@@ -17,6 +17,9 @@ To connect your Android device to a local deployment, you need to prepare its co
 1. Select **Save** and close the window.
 
 ## Unity Editor{#in-editor}
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want. <br> 
+In your Unity Editor, add this script it to your Android client-worker GameObject. In the FPS Starter Project this is located at **Assets** > **Fps** > **Prefabs** > **AndroidClientWorker**.
```

Comment:
```suggestion
In your Unity Editor, add this script to your Android client-worker GameObject. In the FPS Starter Project, navigate to **Assets** > **Fps** > **Prefabs** > **AndroidClientWorker**.
```

Reply:
Included this change.
Also, changed "In your Unity Editor" to "In the Unity Editor" to match the same language used elsewhere in the page.

Reply:
good call! unfortunately I'll have to ask you to change it to "In your Unity Editor" everywhere on the page. We are trying to use "your" everywhere, but missed some places :(

Reply:
All references to "in the Unity Editor" have been replaced with "in your Unity Editor".